mqtt:
  connection:
    host: "YOUR_ENDPOINT_HERE.iot.us-east-1.amazonaws.com"
    port: 8883
    keepalive: 60
  tls:
    ca_certs: "/home/user/catkin_ws/src/mqtt_bridge/config/certs/AmazonRootCA1.pem"
    certfile: "/home/user/catkin_ws/src/mqtt_bridge/config/certs/certificate.pem.crt"
    keyfile: "/home/user/catkin_ws/src/mqtt_bridge/config/certs/private.pem.key"

bridge:
  # Publish Sensor Data to AWS
  
factory: mqtt_bridge.bridge:RosToMqttBridge
  msg_type: sensor_msgs.msg:LaserScan
  topic_from: /scan
  topic_to: robot/telemetry/scan
factory: mqtt_bridge.bridge:RosToMqttBridge
msg_type: nav_msgs.msg:Odometry
topic_from: /odom
topic_to: robot/telemetry/odom

  # Receive Commands from AWS
  
factory: mqtt_bridge.bridge:MqttToRosBridge
  msg_type: geometry_msgs.msg:Twist
  topic_from: robot/commands/cmd_vel
  topic_to: /cmd_vel